vcr/vcr.go

```diff
@@ -140,11 +144,29 @@ func (c *vcr) Configure(config core.ServerConfig) error {
 
 	networkPublisher := issuer.NewNetworkPublisher(c.network, c.docResolver, c.keyStore)
 	if c.config.OIDC4VCI.Enabled {
-		if config.Auth.PublicURL == "" {
-			return errors.New("auth.publicurl is required to enable OIDC4VCI")
+		if c.config.OIDC4VCI.URL == "" {
+			return errors.New("vcr.oidc4vci.url must be configured to enable OIDC4VCI")
 		}
-		c.publicBaseURL = config.Auth.PublicURL
-		c.oidcIssuer = issuer.NewOIDCIssuer(core.JoinURLPaths(c.publicBaseURL, "identity"))
+		// Must be either HTTP or HTTPS, in strict mode HTTPS is required
+		c.baseURL = c.config.OIDC4VCI.URL
+		if strings.HasPrefix(c.baseURL, "http://") {
+			if config.Strictmode {
+				return errors.New("vcr.oidc4vci.url must use HTTPS when strictmode is enabled")
+			}
+		} else if !strings.HasPrefix(c.baseURL, "https://") {
+			return errors.New("vcr.oidc4vci.url must contain a valid URL (using http:// or https://)")
+		}
+
+		var tlsConfig *tls.Config
+		if config.TLS.Enabled() {
+			var err error
+			c.crlValidator, c.clientTLSConfig, err = pki.CreateTLSConfig(config.TLS)
+			if err != nil {
+				return err
+			}
+		}
```

Comment:
Given our [recommended deployment](https://nuts-node.readthedocs.io/en/stable/pages/deployment/recommended-deployment.html) for /n2n, I just want to confirm that this is needed because we have no outgoing proxy?

Reply:
Yes